<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LooseChangePresenter</class>
 <widget class="QMainWindow" name="LooseChangePresenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Loose Change</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="toolButtonOpen">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>10</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Open File</string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset resource="../Icons/Icons.qrc">
      <normaloff>:/OpenIcon.png</normaloff>:/OpenIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonShowRawViewPresenter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>70</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Raw View</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonSave">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>10</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset resource="../Icons/Icons.qrc">
      <normaloff>:/SaveIcon.png</normaloff>:/SaveIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonAddTransaction">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add Transaction [Ctrl+A]</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Icons/Icons.qrc">
      <normaloff>:/PlusIcon.png</normaloff>:/PlusIcon.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionEdit_Categories"/>
    <addaction name="actionAdd_Transaction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../Icons/Icons.qrc">
     <normaloff>:/OpenIcon.png</normaloff>:/OpenIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../Icons/Icons.qrc">
     <normaloff>:/SaveIcon.png</normaloff>:/SaveIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionEdit_Categories">
   <property name="text">
    <string>&amp;Edit Categories</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAdd_Transaction">
   <property name="icon">
    <iconset resource="../Icons/Icons.qrc">
     <normaloff>:/PlusIcon.png</normaloff>:/PlusIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add Transaction</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Icons/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
